clients/src/main/java/org/apache/kafka/common/requests/FetchResponse.java

```diff
@@ -165,7 +165,8 @@ public String toString() {
             return "(error=" + error + ", highWaterMark=" + highWatermark +
                     ", lastStableOffset = " + lastStableOffset +
                     ", logStartOffset = " + logStartOffset +
-                    ", abortedTransactions = " + abortedTransactions + ", records=" + records + ")";
+                    ", abortedTransactions = " + abortedTransactions +
+                    ", recordsSizeInBytes=" + records.sizeInBytes() + ")";
```

Comment:
If possible, let's save that for a later patch. At the moment I would like to capture all of the info in `PartitionData` above. From users, debug logs are usually the best we can get, and all of this information is useful when debugging consumer problems.